documentation/dataplane/dash-flow/DASH_FLOW_SAI.md

```diff
@@ -0,0 +1,615 @@
+
+
+# README
+
+DASH Flow SAI abstracts the flow state within DASH. In the quest for achieving cloud services, the control plane can generate a flow state table, performing operations such as adding, deleting, modifying, and querying. This specification guarantees alignment between the flow state table and flow states across different cloud providers and vendors. We propose an API to provide a vendor-neutral method for managing flow states, including uniform control of Programmable switches, SmartNICs, and Smart Switches.
```

Comment:
flow state seems to have ambiguity in the doc, so will be better to ensure consistency:
- flow: it represents a single direction of the match-action entry for a network connection.
- flow key: the key that used to match the packet for finding its flow
- flow state: the state of the flow, including the packet transformations and all other tracked states, such as TCP, HA and etc.

fixing the wording in the doc and adding a terminology section will be greatly helpful.

Reply:
I believe we can change "DASH Flow SAI" to either "DASH flow" or "DASH flow SAI APIs", depending on the context, whether it means the concept or it means the API,

Reply:
same for the rest of the doc.